### PR TITLE
PAL-569, updated copyrights within the Palisade Service

### PR DESCRIPTION
**Pull request by developer01189998819991197253**

### CODE REVIEW

File: audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/ContractTestData.java [hunk 1]
```diff
@@ -51,9 +51,9 @@ private ContractTestData() {
...
-    public static final String BAD_SUCCESS_REQUEST_JSON = "{\"userId\":\"test-user-id\",\"resourceId\":\"/test/resourceId\",\"context\":{\"class\":\"uk.gov.gchq.palisade.Context\",\"contents\":{\"purpose\":\"purpose\"}},\"serviceName\":\"user-service\",\"timestamp\":\"2020-01-01\",\"serverIP\":\"The IP of the server\",\"serverHostname\":\"The name of the server\",\"attributes\":{\"messages\":\"5\"},\"leafResourceId\":\"file:/test/resource/file.txt\"}";
+    public static final String ERROR_REQUEST_JSON = "{\"userId\":\"test-user-id\",\"resourceId\":\"/test/resourceId\",\"context\":{\"class\":\"uk.gov.gchq.palisade.Context\",\"contents\":{\"purpose\":\"purpose\"}},\"serviceName\":\"user-service\",\"timestamp\":\"2018-2021-01-01\",\"serverIP\":\"The IP of the server\",\"serverHostname\":\"The name of the server\",\"attributes\":{\"messages\":\"5\"},\"error\":{\"cause\":null,\"stackTrace\":[],\"message\":\"exception message\",\"suppressed\":[],\"localizedMessage\":\"exception message\"}}";
```
Comment (kg981167224) [inline audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/ContractTestData.java]: This section doesn't look correct.

File: audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/ContractTestData.java [hunk 2]
```diff
@@ -51,9 +51,9 @@ private ContractTestData() {
...
+    public static final String GOOD_SUCCESS_REQUEST_JSON = "{\"userId\":\"test-user-id\",\"resourceId\":\"/test/resourceId\",\"context\":{\"class\":\"uk.gov.gchq.palisade.Context\",\"contents\":{\"purpose\":\"purpose\"}},\"serviceName\":\"data-service\",\"timestamp\":\"2018-2021-01-01\",\"serverIP\":\"The IP of the server\",\"serverHostname\":\"The name of the server\",\"attributes\":{\"messages\":\"5\"},\"leafResourceId\":\"file:/test/resource/file.txt\"}";
+    public static final String BAD_SUCCESS_REQUEST_JSON = "{\"userId\":\"test-user-id\",\"resourceId\":\"/test/resourceId\",\"context\":{\"class\":\"uk.gov.gchq.palisade.Context\",\"contents\":{\"purpose\":\"purpose\"}},\"serviceName\":\"user-service\",\"timestamp\":\"2018-2021-01-01\",\"serverIP\":\"The IP of the server\",\"serverHostname\":\"The name of the server\",\"attributes\":{\"messages\":\"5\"},\"leafResourceId\":\"file:/test/resource/file.txt\"}";
```
Comment (kg981167224) [inline audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/ContractTestData.java]: JSON timestamp looks wrong

File: audit-service/README.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ curl -X POST api/audit -H "content-type: application/json" --data \
       }
    },
    "serviceName":"testServicename",
-   "timestamp":"2020-11-26T10:28:54.118082Z",
+   "timestamp":"2018-2021-11-26T10:28:54.118082Z",
```
Comment (kg981167224) [inline audit-service/README.md]: Timestamp isn't valid

File: audit-service/README.md [hunk 4]
```diff
@@ -105,7 +105,7 @@ curl -X POST api/audit -H "content-type: application/json" --data \
       }
    },
    "serviceName":"testServicename",
-   "timestamp":"2020-11-26T10:32:28.818842Z",
+   "timestamp":"2018-2021-11-26T10:32:28.818842Z",
```
Comment (kg981167224) [inline audit-service/README.md]: Timestamp isn't valid

**Review by kg981167224 — CHANGES_REQUESTED** [review]
Think it needs some changes.

File: audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/ContractTestData.java (hunk 1 above)
Comment (developer01189998819991197253) [reply audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/ContractTestData.java]: reset, good spot

File: audit-service/README.md (hunk 3 above)
Comment (developer01189998819991197253) [reply audit-service/README.md]: reset

File: audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/ContractTestData.java (hunk 2 above)
Comment (developer01189998819991197253) [reply audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/ContractTestData.java]: reset

File: audit-service/README.md (hunk 4 above)
Comment (developer01189998819991197253) [reply audit-service/README.md]: reset

**Review by kg981167224 — APPROVED** [review]
Can't see anything missed.

**Review by dev930018 — APPROVED** [review]

**Pull request merged**